### DISCUSSION TITLE
Is there support for other commercial LLMs such as Cohere, Claude (or other models from HF) ?

### DISCUSSION THREAD

**Discussion by yash-srivastava19** [crewAIInc/crewAI#1130] [Q&A]
I wanted to ask whether CrewAI supports models such as Cohere/Claude or the ones from Huggingface as I'm unable to find those. If you are planning to work on that, how exactly would that pan out? (If the issue is in open, I would like to work on it)

Thanks !

**Comment by Biancamazzi** [comment]
don't forget we have https://chat.openai.com/g/g-qqTuUWsBY-crewai-assistant to help and also the discord community https://discord.com/invite/X4JWnZnxPb

**Comment by C-1pher** [comment]
CrewAI use Langchain for this so yo can call any llm provider if LangChain support it , the one you mentioned work with LangChain so it works with CrewAI out of the box
